me.drewol.Unnamed-SDVX-Clone.yml

```diff
@@ -0,0 +1,53 @@
+id: me.drewol.Unnamed-SDVX-Clone
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: usc-game
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # GPU acceleration
+  - --device=dri
+  # Needs to talk to the network
+  - --share=network
+  # Sound output
+  - --socket=pulseaudio
+modules:
+  - name: unnamed-sdvx-clone-app
+    buildsystem: simple
+    sources:
+      - type: file
+        path: me.drewol.Unnamed-SDVX-Clone.desktop
```

Comment:
this is a .desktop upstream so there is no reason to have this here

https://github.com/Drewol/unnamed-sdvx-clone/blob/develop/appimage/usc-game.desktop